jsonnetsecure/jsonnet_pool.go

```diff
@@ -203,6 +208,8 @@ func (vm *processPoolVM) EvaluateAnonymousSnippet(filename string, snippet strin
 		return "", errors.Wrap(err, "jsonnetsecure: acquire")
 	}
 
+	ctx, cancel := context.WithTimeout(ctx, 1*time.Second)
```

Comment:
Why should this not be configurable?

Reply:
It is: by passing the context to `func (p *DefaultProvider) JsonnetVM(ctx context.Context) (VM, error)` you can choose any timeout you want.

This one-second cooldown is just a hard limit for the evaluation. I've never seen the actual evaluation step taking more than 10ms, except with adversarial workloads.